FF1RandomizerOnline/wwwroot/js/site.js

```diff
@@ -117,6 +121,17 @@ function setCallbacks() {
 	};
 }
 
+function OnFlagsChanged() {
```

Comment:
Make this camelCase and we're good to go.

Reply:
We should all agree that kebab-case in a sea of parentheses is the superior choice and be done with it. :D